test/stripe/converter_test.exs

```diff
@@ -0,0 +1,128 @@
+defmodule Stripe.ConverterTest do
+  use ExUnit.Case
+
+  alias Stripe.Converter
+
+  defmodule Person do
+    defstruct [:email, :first_name, :last_name, :legal_entity, :metadata]
+    def relationships, do: %{}
+  end
+
+  defmodule PersonWithMissingRelationship do
+    defstruct [:card]
+    def relationships, do: %{card: Stripe.ConverterTest.CreditCard}
+  end
```

Comment:
I've never seen defining modules with structs like this in a test. This is handy to know! 👍